### PR TITLE
Converts /artist-series/:slug to v3

### PR DESCRIPTION
**Pull request by dzucconi**

### CODE REVIEW

File: src/v2/Apps/Artist/Components/ArtistSeriesRail/V2ArtistSeriesItem.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+} from "v2/Artsy/Analytics/AnalyticsContext"
+
+interface Props extends AnalyticsContextProps {
+  artistSeries: ArtistSeriesItem_artistSeries
+  lazyLoad: boolean
+  contextModule: ContextModule
+  index: number
+}
+
+export const V2ArtistSeriesItem: React.FC<Props> = props => {
```
Comment (dzucconi) [inline src/v2/Apps/Artist/Components/ArtistSeriesRail/V2ArtistSeriesItem.tsx:21]: See comment https://github.com/artsy/force/compare/dzucconi/refactor/artist-series?expand=1#diff-253b08e9b07e84621b1c3c8b7be042b104a482578e675dae4a39a23788247188R15-R19

File: src/v2/Apps/Artist/Components/ArtistSeriesRail/V2ArtistSeriesRail.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  artist: ArtistSeriesRail_artist
+  title?: string
+  contextModule: ContextModule
+}
+
+/**
+ * TODO: When this page is migrated to V3:
+ * This component can be deleted and replaced with the V3 ArtistSeriesRail in
+ * v2/Components/ArtistSeriesRail
+ */
```
Comment (dzucconi) [inline src/v2/Apps/Artist/Components/ArtistSeriesRail/V2ArtistSeriesRail.tsx:19]: This component was being used in a v2 context here on the artist page. So I copied the old one over into the artist directory.

File: src/v2/Apps/ArtistSeries/Components/ArtistSeriesHeader.tsx [hunk 3]
```diff
@@ -1,225 +1,128 @@
...
-        <Flex alignItems="center" justifyContent="center" position="relative">
-          <Flex position="absolute" left={0}>
-            {/* @ts-expect-error STRICT_NULL_CHECK */}
-            {artists.length && (
-              <ArtistInfo
-                contextOwnerId={internalID}
-                contextOwnerSlug={slug}
-                // @ts-expect-error STRICT_NULL_CHECK
-                artist={artists[0]}
+      <TopContextBar>
```
Comment (dzucconi) [inline src/v2/Apps/ArtistSeries/Components/ArtistSeriesHeader.tsx:42]: Worked with Jonathan to figure this one out — So now we've got a common `TopContextBar` — we'll probably be adding this back to the Show page and a few others.

Comment (damassi) [reply src/v2/Apps/ArtistSeries/Components/ArtistSeriesHeader.tsx:42]: Mind throwing a screenshot of what this is here?

**Review by damassi — APPROVED** [review]

File: src/v2/Apps/ArtistSeries/Components/ArtistSeriesHeader.tsx (hunk 3 above)
Comment (dzucconi) [reply src/v2/Apps/ArtistSeries/Components/ArtistSeriesHeader.tsx:42]: Just this — Very minor thing.

**Pull request merged**